lib/fs.js

```diff
@@ -1617,8 +1617,15 @@ function ReadStream(path, options) {
   if (!(this instanceof ReadStream))
     return new ReadStream(path, options);
 
+  if (options === undefined)
```

Comment:
Current versions accept `null` and any other falsey value so this is still a breaking change. If this were to land, it would make sense to throw on `null`, since this is already a breaking change and `null` is not a useful value to support here. This needs to be given a lot of consideration, especially after the events of this past weekend.